tools/branch_comparisons.md

```diff
@@ -0,0 +1,93 @@
+# Branch comparisons
+
+These links take to comparisons between branches.
+
+When it says "There isn’t anything to compare," this means no pull requests
+need to be forward-ported.
```

Comment:
```suggestion
If a cell in the table is `-`, this means that the two Ignition releases use the same version of this Ignition library.
So, for example, `ign-cmake` has a `-` for B ➡️  C because Blueprint and Citadel use `ign-cmake2`.

When it says "There isn’t anything to compare," this means no pull requests
need to be forward-ported.
```

Reply:
824176b170a8353a14f69bde100af7982fd8aa53